modules/material-management/src/types.ts

```diff
@@ -65,3 +65,27 @@ export type DecryptionMaterial<Suite> =
   Suite extends NodeAlgorithmSuite ? NodeDecryptionMaterial :
   Suite extends WebCryptoAlgorithmSuite ? WebCryptoDecryptionMaterial :
   never
+
+export type AwsEsdkKeyObjectType = 'secret' | 'public' | 'private'
```

Comment:
What is the purpose of these exports?

Reply:
Need to add comments.  Sorry.  This is because there are different versions of Node.js.  If I use/export the v11/v12 version then Typescript will complain if consumers of this package use any other version of Node.js (and we must support older versions.)